### PR TITLE
fix: reset line height and font weight for tab titles

### LINKED ISSUE #14968: [Bug]: Missing border css for debugger panel in JSObject
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Border to be added above JSObject debugger panel.
Earlier(This is how it should be)
Screenshot 2022-06-29 at 7 05 06 PM
Now
Screenshot 2022-06-29 at 7 04 52 PM

### Steps To Reproduce

Go to JSObject panel

### Public Sample App

_No response_

### Version

Release 1.7.6

### PR DESCRIPTION
**Pull request by yaldram**
## Description
- The bottom border was not visible for the js editor, also the tab titles in the js editor, api editor did not have a bold font weight as they did previously.

Fixes #14968 #14981

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Tested Manually on the canvas.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ChandanBalajiBP — DISMISSED** [review]

**Review by ChandanBalajiBP — DISMISSED** [review]

File: app/client/src/components/ads/Tabs.tsx [hunk 1]
```diff
@@ -41,8 +41,7 @@ const TabsWrapper = styled.div<{
     flex-direction: ${(props) => (!!props.vertical ? "column" : "row")};
     align-items: ${(props) => (!!props.vertical ? "stretch" : "center")};
     border-bottom: none;
-    gap: ${(props) =>
-      !props.vertical ? `${props.theme.spaces[12] + 2}px` : 0};
+    gap: ${(props) => (!props.vertical ? `${props.theme.spaces[12]}px` : 0)};
```
Comment (albinAppsmith) [inline app/client/src/components/ads/Tabs.tsx:44]: Make sure this won't affect the exiting places.

**Review by albinAppsmith — DISMISSED** [review]

**Comment by ChandanBalajiBP** [comment]
6 ge 45ees4d678 6f fxfs TX y x fxfs, 4 5 cc B x3 amaskaara,, 55 c. S set c
db TX dxg yes

File: app/client/src/components/ads/Tabs.tsx [hunk 2]
```diff
@@ -78,11 +76,9 @@ const TabsWrapper = styled.div<{
     position: relative;
     padding: 0;
```
Comment (ChandanBalajiBP) [inline app/client/src/components/ads/Tabs.tsx:77]: can you please remove this padding. As we have overriding padding below.

File: app/client/src/components/ads/Tabs.tsx [hunk 3]
```diff
@@ -78,11 +76,9 @@ const TabsWrapper = styled.div<{
...
 
-    ${(props) =>
-      props.responseViewer &&
-      `
-       
-      `}
```
Comment (ChandanBalajiBP) [inline app/client/src/components/ads/Tabs.tsx:85]: This was unnecessary piece of code. Thank you for removing this.

**Review by ChandanBalajiBP — APPROVED** [review]

**Pull request merged**